boutdata/squashoutput.py

```diff
@@ -140,6 +144,11 @@ def squashoutput(
         oldfile = datadirnew + "/" + outputname
         datadir = datadirnew
 
+    if drop_variables is None:
```

Comment:
```suggestion
    if not drop_variables:
```
This would also catch if `drop_variables=False` for some reason ...